vuepress/docs/next/tutorials/create/mfe/react.md

```diff
@@ -1,169 +1,123 @@
 
-2. In `public/index.html`, replace `<div id="root"></div>` with this:
+   export default WidgetElement;
+   ```
 
-``` html
-    <my-widget />
-```
+   ::: tip Custom Element Names
 
-3. Observe your browser automatically redisplay the React app
+   - [Must contain a hyphen `-` in the name](https://stackoverflow.com/questions/22545621/do-custom-elements-require-a-dash-in-their-name)
+   - Cannot be a single word
+   - Should follow `kebab-case` naming convention
+   :::
 
-::: tip Congratulations!
-You’re now using a custom element to display a React app.
-:::
-
-## Display the Micro Frontend in Entando
+### Display the Custom Element
 
-### Build the React App
+1. Replace the entire contents of `src/index.js` with these two lines: 
+   ``` js
+   import './index.css';
+   import './WidgetElement';
+   ```
 
-To deploy the custom element into Entando as a micro frontend, you must perform a production build of the React app.
+2. In `public/index.html`, replace `<div id="root"></div>` with the following:
+   ``` html
+       <simple-mfe />
+   ```
 
-1. In the `my-widget` directory, create an `.env.production` file that consists of one line:
-``` text
-PUBLIC_URL=/entando-de-app/cmsresources/my-widget
-```
+3. Observe your browser automatically redisplay the React app.
 
-::: warning Notes
-- `/entando-de-app/cmsresources/` is the Resource URL for your Entando Application, which matches nearly all development environments. Consult your Entando admin or use the following fragment to discover the Resource URL.
-``` ftl 
-  <#assign wp=JspTaglibs[ "/aps-core"]>
-  <@wp.resourceURL />
-``` 
-- `/my-widget` is the public folder that hosts the JavaScript and CSS files for the React app
+::: tip Congratulations!
+You’re now using a custom element to display a React app.
 :::
 
+## Display the React MFE in Entando
 
-2. Build the app:
-``` bash
-npm run build
-```
-
-### Upload the React files
-To set up the micro frontend to use in Entando:
-
-1. In your App Builder, go to `Administration` → `File browser` → `public`
-
-2. Click `Create folder` and name it "my-widget" to match the `.env.production` path above
-
-3. Click `Save`
+1. From the root bundle directory, generate the Docker image:
+   ``` sh
+   ent bundle pack
+   ```
+   This builds the widget and constructs a Docker image for the bundle.
 
-4. Click `my-widget`
+2. Publish the Docker image to a Docker registry:
+   ``` sh
+   ent bundle publish
+   ```
 
-5. Create a folder structure similar to your generated build directory:
+3. Deploy the bundle to your Entando Application:
+   ``` sh
+   ent bundle deploy
+   ```
+   Your bundle will appear in the Local Hub of your Entando instance, accessible from `App Builder` → `Hub`, and show a status of DEPLOYED.
 
-- `my-widget/static/css`
-- `my-widget/static/js`
-- `my-widget/static/media`
+4. Install the bundle in your Entando Application from `App Builder` → `Hub` or with the following command:
```

Comment:
this is awkward and confusing. It also reads like "install the bundle in your entando application from app builder to the Hub" I would break it up or edit